### PR TITLE
[Docs]- Fix Flickering in Links 

### LINKED ISSUE #9196: [Docs]- Flickering in Links
### Current Behavior

 Links that flickers in https://docs.meshery.io/ -
- https://docs.meshery.io/installation/mesheryctl
- https://docs.meshery.io/installation/compatibility-matrix
- https://docs.meshery.io/project/contributing

### Expected Behavior

There should be no occurrence of link flickering."

### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by coder12git**
**Notes for Reviewers**

This PR fixes #9196

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 94d333d83b9a03d2775a12e6b6f70843f617c363 at: https://65391b77e54a6c1c5a574561--meshery-docs-preview.netlify.app

**Comment by coder12git** [comment]
Links that flickers in https://docs.meshery.io/ -

- https://docs.meshery.io/installation/mesheryctl
- https://docs.meshery.io/installation/compatibility-matrix
- https://docs.meshery.io/project/contributing

**Comment by coder12git** [comment]
// @vishalvivekm

**Review by abhijeetgauravm — COMMENTED** [review]
Looks good to me!

**Review by sudhanshutech — DISMISSED** [review]

**Comment by sudhanshutech** [comment]
@coder12git merge conflict

File: docs/pages/installation/mesheryctl.md [hunk 1]
```diff
@@ -4,9 +4,6 @@ title: Install mesheryctl
 permalink: installation/mesheryctl
 type: installation
 category: mesheryctl
-redirect_from:
- - installation/mesheryctl
- - installation/mesheryctl/
```
Comment (leecalcote) [inline docs/pages/installation/mesheryctl.md:9]: Thanks. We need to keep the redirect that has the trailing `/`.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/pages/installation/mesheryctl.md [hunk 2]
```diff
@@ -4,9 +4,6 @@ title: Install mesheryctl
 permalink: installation/mesheryctl
 type: installation
 category: mesheryctl
```
Comment (leecalcote) [inline docs/pages/installation/mesheryctl.md:6]:
```suggestion
category: mesheryctl
redirect_from:
 - installation/mesheryctl/
```

File: docs/pages/project/compatibility_matrix/compatibility.md [hunk 3]
```diff
@@ -2,10 +2,6 @@
 layout: default
 title: Compatibility Matrix
 permalink: installation/compatibility-matrix
```
Comment (leecalcote) [inline docs/pages/project/compatibility_matrix/compatibility.md:4]:
```suggestion
permalink: installation/compatibility-matrix
redirect_from: 
 - project/compatibility-matrix/
 - installation/compatibility-matrix/
```

File: docs/pages/project/contributing/contributing.md [hunk 4]
```diff
@@ -3,9 +3,6 @@ layout: page
 title: Contributing
 permalink: project/contributing
 description: How to contribute to the Meshery project.
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing.md:5]:
```suggestion
description: How to contribute to the Meshery project.
redirect_from:
 - project/contributing/
```

**Pull request merged**